jekyll/_cci2/language-ruby.md

```diff
@@ -309,7 +304,10 @@ This app illustrates the simplest possible setup for a Ruby on Rails web app. Re
 * [Discourse](https://github.com/CircleCI-Public/discourse/blob/master/.circleci/config.yml), an open source discussion platform.
 * [Sinatra](https://github.com/CircleCI-Public/circleci-demo-ruby-sinatra), a demo app for the [simple DSL for quickly creating web applications](http://www.sinatrarb.com/).
 
-If you have any questions, head over to our [community forum](https://discuss.circleci.com/) for support from us and other users.
+## See Also
+
+The [Linux Project Tutorial](#config-walkthrough) also provides a detailed explanation of CircleCI 2.0 configuration.
```

Comment:
This links to the Config Walkthrough section of this document.

Reply:
thanks, removed